src/main/car/inventory/Dealership.java

```diff
@@ -1,57 +1,178 @@
-        return dealerID;
-    }
-
-    public boolean isVehicleAcquisition() {
-        return vehicleAcquisition;
-    }
-
-    public Map<String, Vehicle> getVehicleInventory() {
-        return vehicleInventory;
-    }
 
     public Dealership(String newDealer) {
         dealerID = newDealer;
         vehicleAcquisition = true;
         vehicleInventory = new HashMap<>();
     }
-    //constructors
+
+    // constructors
     public Dealership(String dealerID, boolean vehicleAcquisition, HashMap<String, Vehicle> vehicleInventory) {
         this.dealerID = dealerID;
         this.vehicleAcquisition = vehicleAcquisition;
         this.vehicleInventory = vehicleInventory;
     }
-    //methods
+
+    // methods
     public void addIncomingVehicle(String stockNumber, Vehicle car) {
-        if(car.getVehicleType().equalsIgnoreCase("SUV") || car.getVehicleType().equalsIgnoreCase("Sedan") || car.getVehicleType().equalsIgnoreCase("Pickup") || car.getVehicleType().equalsIgnoreCase("Sports Car")) {
-            if(vehicleAcquisition == true && this.vehicleInventory.containsKey(stockNumber) == false){
+        if (car.getVehicleType().equalsIgnoreCase("SUV") || car.getVehicleType().equalsIgnoreCase("Sedan")
+                || car.getVehicleType().equalsIgnoreCase("Pickup")
+                || car.getVehicleType().equalsIgnoreCase("Sports Car")) {
+            if (vehicleAcquisition == true && this.vehicleInventory.containsKey(stockNumber) == false) {
                 this.vehicleInventory.put(stockNumber, car);
-            } else if(vehicleAcquisition == false){
-                System.out.println("Dealer " + this.getDealerID() + " is not allowed to add additional vehicles to their inventory\n");
-            } else if(this.vehicleInventory.containsKey(stockNumber) == true) {
+            } else if (vehicleAcquisition == false) {
+                System.out.println("Dealer " + this.getDealerID()
+                        + " is not allowed to add additional vehicles to their inventory\n");
+            } else if (this.vehicleInventory.containsKey(stockNumber) == true) {
                 System.out.println("Vehicle " + car.getVehicleID() + " is already in this dealer's inventory\n");
             }
         } else {
             System.out.println("Not a valid vehicle type.\n");
         }
     }
 
+    public void clearInventory() {
+        vehicleInventory.clear();
+    }
+
+    // check if a specific car is rented or not BUT RETURN BOOLEAN
+    public boolean checkIfCarIsRented(Vehicle car) {
+        boolean rentedOrNot = false;
+        for (Map.Entry<String, Vehicle> vehicle : vehicleInventory.entrySet()) {
+            if(car.getVehicleID().equalsIgnoreCase(vehicle.getValue().getVehicleID())) {
+                rentedOrNot = car.getRentStatus();
+            }
+        }
+        return rentedOrNot;
+    }
+
+    // check if a specific car is rented or not BUT RETURN STRING YES OR NO
+    public String isCarIsRented(Vehicle car) {
```

Comment:
this method, isCarIsRented() is not required, plus checkIfCarIsRented() already does the same thing. Also the previous method would be better named isVehicleRented() rather than checkIfCarIsRented().